components/auth/SignOut.tsx

```diff
@@ -2,11 +2,17 @@ import type { ButtonProps } from "react-bootstrap"
 import { Button } from "../bootstrap"
 import { auth } from "../firebase"
 
+// Signout, and redirect to the home page.
+const handleSignout = () => {
```

Comment:
Let's rename this signOutAndRedirectToHome to be explicit, move it into `auth/service.tsx`, and share it between SignOut and layout.

Reply:
I have updated this on the latest commit.
Thank you